firmware/thruna/PDM/src/app/states/app_initState.c

```diff
@@ -17,6 +21,14 @@ static void initStateRunOnTick1Hz(struct StateMachine *const state_machine)
 static void initStateRunOnTick100Hz(struct StateMachine *const state_machine)
 {
     app_allStates_runOnTick100Hz(state_machine);
+
+    bool isInverterOnOrDriveState =
+        App_CanRx_BMS_State_Get() == BMS_INVERTER_ON_STATE || App_CanRx_BMS_State_Get() == BMS_DRIVE_STATE;
+
+    if (isInverterOnOrDriveState)
+    {
+        App_SharedStateMachine_SetNextState(state_machine, app_driveState_get());
+    }
```

Comment:
Can you also get precharge state into this list?

nit: snake case pls `inverter_on_or_drive_state`